src/ui/czi-ordered-list.css

```diff
@@ -1,18 +1,31 @@
+html {
+  /* This sets up the initial counter values to `0`. */
+  counter-reset:
+    czi-counter-0 czi-counter-1 czi-counter-2 czi-counter-3
+    czi-counter-4 czi-counter-5 czi-counter-6 czi-counter-7;
```

Comment:
why does this go specifically to 7?

Reply:
This is defined at `src/ParagraphNodeSpec.js`
I'll add CSS comments to document this.

![image](https://user-images.githubusercontent.com/1504439/58358293-2b83ca80-7e33-11e9-94ba-f8af3cf5b40a.png)